Dockerfile

```diff
@@ -33,22 +33,22 @@ RUN apt-get update \
     python3-pip \
   && rm -rf /var/lib/apt/lists/*
 
-# Install protoc 3.5.1.
+# Install protoc 3.6.0.
 # Currently, the stable version is still 2.6.x, which can only handle proto2
```

Comment:
Is this old comment useful or relevant?

Reply:
I suspect it's not (certainly wasn't to me)

Reply:
Maybe remove it now?